src/engine/meta-plugin.ts

```diff
@@ -4,9 +4,9 @@ import { warn } from '../cli'
 
 export default function metaPlugin(_, marpit: Marpit) {
   Object.assign(marpit.customDirectives.global, {
-    description: v => ({ marpCLIDescription: v }),
-    image: v => ({ marpCLIImage: v }),
-    title: v => ({ marpCLITitle: v }),
+    description: v => (typeof v === 'string' ? { marpCLIDescription: v } : {}),
```

Comment:
Marpit v0.7.1 has updated type definitions to allow array/object passing to custom directives. (marp-team/marpit#130) We have to validate whether meta value is string.